GVFS/GVFS.FunctionalTests/Tests/FastFetchTests.cs

```diff
@@ -476,16 +493,24 @@ private ProcessResult RunFastFetch(string args)
         {
             args = args + " --verbose";
 
-            string fastfetch = Path.Combine(Settings.Default.CurrentDirectory, "fastfetch.exe");
-            if (!File.Exists(fastfetch))
-            {
-                fastfetch = "fastfetch.exe";
+            string fastfetch;
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                fastfetch = Path.Combine(Settings.Default.CurrentDirectory, "netcoreapp2.1", "fastfetch.dll");
+                if (!File.Exists(fastfetch))

```

Comment:
Do you know if this check is still needed? I can't remember why we'd fall back on "fastfetch.exe", @sanoursa, do you have any ideas?

Reply:
Check out mseng PR#269997 for the PR where this change was introduced. From the comments there, it looks like this was to work around issues in the Windows build lab. Since you left the comments on this code there, I'll defer to your good judgment on if this is now dead code. :)